conformance/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -97,6 +97,15 @@ object DynamicConformanceJob {
     processResult(result, performance, publishPath, stdPath)
   }
 
+  private def isExperimentalRuleEnabled()(implicit cmd: CmdConfig): Boolean = {
+    val enabled = cmd.experimentalMappingRule match {
+      case Some(b) => b
+      case None => conf.getBoolean("conformance.mapping.rule.experimental.implementation")
```

Comment:
Does this explode when its not provided as a cli argument and missing in props?

Reply:
Yeah, it can. Will make it more safe.

Reply:
I had the same thought, but if we are providing default in the properties, it should be ok to leave it.

Reply:
This is fixed. Merging...